### DISCUSSION TITLE
Create static thumbnails of animated GIFs

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3121] [Feature Requests]
Currently, GIFs are shown as-is in the File Library. This means that you could be loading a LOT of data (eg: 1-3MB each x100+ items) on the browse page. We should find a way to render a static thumbnail for these. Animated GIFs can just be the first frame.

**Comment by rijkvanzanten** [comment]
> We should find a way to render a static thumbnail for these.

Ideally, the asset endpoint can actually resize the thumbnails instead of just return a static image

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
